Source/Core/Core/IOS/USB/Emulated/Skylander.h

```diff
@@ -0,0 +1,102 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <map>
+#include <mutex>
+#include <queue>
+#include <vector>
+
+#include "Common/IOFile.h"
+#include "Core/IOS/USB/EmulatedUSBDevice.h"
+
+// The maximum possible characters the portal can handle.
+// The status array is 32 bits and every character takes 2 bits.
+// 32/2 = 16
+constexpr u8 MAX_SKYLANDERS = 16;
+
+namespace IOS::HLE::USB
```

Comment:
You might want to add an extra namespace here, it's kinda weird for eg. `LedColor` to just hang out in the USB namespace. Or maybe rename every class to specifically refer to the Skylander USB device.

Reply:
Have changed the class name to SkylanderLEDColor for now